### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by pakkurthi** [hey-its-d2t2/Job-Portal#1]
Hey,
There are many errors in controller ,services and util .
Please go through them . 
![image](https://github.com/user-attachments/assets/6b05756e-4694-4a55-838a-216f00f1642a)

**Comment by hey-its-d2t2** [comment]
Thank you for pointing out the issues, Pakkurthi. Currently, I’m not actively working on this project, but I’ll make sure to address the errors when I get the chance. In the meantime, if you’ve already fixed any of them, please feel free to raise a PR. I’d greatly appreciate your contribution and collaboration. Thanks again for taking the time to review and share your feedback!

**Comment by Karankk21** [comment]
Hey! Did you correct the errors??

**Comment by hey-its-d2t2** [comment]
> Hey! Did you correct the errors??

Yes, a small  bug related to job listing display and job edit functionality have been fixed. The job titles, company names, and location data now render correctly, and editing job posts works as expected

**Comment by hey-its-d2t2** [comment]
> Hey, There are many errors in controller ,services and util .

Install Maven properly, it seems like Lombok errors

**Comment by HetaleePWarad** [comment]
Hey, does this have all the files, is any file missing in this ?

Can you please re share the file where there are no issues

**Comment by HetaleePWarad** [comment]
One more...if I don't use your summernote files for frontend will there be a issue ?

**Comment by HetaleePWarad** [comment]
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/222dab43-4b03-42d0-b909-2e3099f50c4c" />
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/6a2bed22-2963-4af2-a145-6aa43a03f287" />
I am getting this when i open job seeker dashboard and click edit profile and view saved jobs

**Comment by hey-its-d2t2** [comment]
> Hi HetaleePWarad,
Just pushed some updates added candidate job search & filters, recruiters can view profiles and download resumes now, candidates can save/apply for jobs and view saved ones, plus candidate profile editing. Let me know if you wanna check it out..

**Comment by HetaleePWarad** [comment]
Thankyou for the updated repository.
